content/usage/services.md

```diff
@@ -94,6 +94,10 @@ compose:
 
 For security reasons this option is only available to trusted repositories. Trusted repositories are enabled on a per-repository basis by a Drone administrator from your repository settings screen.
 
+Alternatively, each service will have access to the current source code under the ```/drone``` directory
```

Comment:
I updated the above comment -- the markdown cutoff some of the text I wrote which included a reference to the full path. The full path itself varies and can be customized. I used the `<repository>` placeholder value to signify this